components/src/core/list-item-tag/list-item-tag.tsx

```diff
@@ -34,10 +34,17 @@ const listItemTagStyles = (
 }> =>
     StyleSheet.create({
         root: {
-            backgroundColor: props.backgroundColor || theme.colors.primaryBase || theme.colors.primary,
+            backgroundColor:
+                props.backgroundColor ||
+                (theme.dark ? theme.colors.primaryPalette?.dark : theme.colors.primaryPalette?.main) ||
```

Comment:
I've seen this logic in a few places...maybe we should make a reusable shared function for it

```const get500(theme: Theme): string | undefined => ...logic...```